README.md

```diff
@@ -9,8 +9,11 @@ popularized by Kubernetes.  Libpod also contains the Pod Manager tool `(Podman)`
 * [Continuous Integration:](contrib/cirrus/README.md) [![Build Status](https://api.cirrus-ci.com/github/containers/libpod.svg)](https://cirrus-ci.com/github/containers/libpod/master)
 * [GoDoc: ![GoDoc](https://godoc.org/github.com/containers/libpod/libpod?status.svg)](https://godoc.org/github.com/containers/libpod/libpod)
 * Automated continuous release downloads (including remote-client):
-  * Master Branch: [https://storage.cloud.google.com/libpod-master-releases/](https://storage.cloud.google.com/libpod-master-releases/)
-  * Pull-requests: [https://storage.cloud.google.com/libpod-pr-releases/](https://storage.cloud.google.com/libpod-pr-releases/)
+  * [Latest remote client for Windows](https://storage.googleapis.com/libpod-master-releases/podman-remote-latest-master-windows-amd64.msi)
+  * [Latest remote client for MacOS](https://storage.googleapis.com/libpod-master-releases/podman-remote-latest-master-darwin-amd64.zip)
+  * [Latest Snap package](https://snapcraft.io/podman)
+  * Master Branch Index: [https://storage.googleapis.com/libpod-master-releases/](https://storage.googleapis.com/libpod-master-releases/) - append filename (`<Key>`) to URL to download any file.
```

Comment:
Maybe it's just me, but I grabbed the first key on the list(ibpod-master-releasesfalselibpod-master-fedora-29-amd64.tar.gz156952324178929412019-09-26T18:40:41.788Z"143c9fde4bd33b69c6eef03532a28f42"20776943libpod-master-fedora-29-), appended it to the url and hit a NoSuchKey message.  I shortened the key to the stuff before the quote in the middle, same thing.

Reply:
Whoa, lots of extra stuff, should only be `libpod-master-fedora-29-amd64.tar.gz` as the filename.  

So complete URL would be:  `https://storage.googleapis.com/libpod-master-releases/libpod-master-fedora-29-amd64.tar.gz`.

Hmmm, maybe I can word that better...

Reply:
I just clicked on the link and thought each line was the key (nothing telling me otherwise), so I grabbed it and it didn't go well.